### PR TITLE
Feature coverage

### PR DESCRIPTION
**Pull request by WalterMaier**
Just a general code clean up.
Added regression for Ducted Fan and NACA airfoil geometry for coverage.

### CODE REVIEW

File: trunk/SUAVE/Methods/Propulsion/rayleighflow.py [hunk 1]
```diff
@@ -10,7 +10,10 @@
 #  rayleigh
 # ----------------------------------------------------------------------
 
+<<<<<<< HEAD
```
Comment (planes) [inline trunk/SUAVE/Methods/Propulsion/rayleighflow.py]: Unmerged

Comment (WalterMaier) [reply trunk/SUAVE/Methods/Propulsion/rayleighflow.py]: well that's weird

File: regression/scripts/AVL/avl_files/aircraft.avl [hunk 2]
```diff
@@ -1,167 +0,0 @@
-cruise
```
Comment (planes) [inline regression/scripts/AVL/avl_files/aircraft.avl]: Why is this AVL script deleted?

File: regression/scripts/DC_10_noise/DC_10_noise.py [hunk 3]
```diff
@@ -18,17 +18,17 @@ def main():
     noise = Correlations.shevell(weight_landing, number_of_engines, thrust_sea_level, thrust_landing)
     
     actual = Data()
-    actual.takeoff = 99.7
-    actual.side_line = 97.2
-    actual.landing = 105.2
+    actual.takeoff = 99.982372547196633
```
Comment (planes) [inline regression/scripts/DC_10_noise/DC_10_noise.py]: Hmm, I think these values actually come from Shevell data as a test case. I think what you have done is right. However, we should save the original values in comments somewhere.

File: regression/scripts/atmosphere/constant_temperature.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
+# constnat_temperature.py
```
Comment (planes) [inline regression/scripts/atmosphere/constant_temperature.py]: Typo

File: regression/scripts/geometry/fuselage_planform_compute.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# NACA_airfoil_compute.py
```
Comment (planes) [inline regression/scripts/geometry/fuselage_planform_compute.py]: Name

File: trunk/SUAVE/Analyses/Stability/AVL.py [hunk 6]
```diff
@@ -445,7 +445,12 @@ def build_surrogate(self):
                 CM_sur[ii,jj]    = cm_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
                 Cm_a_sur[ii,jj]  = cm_alpha_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
                 Cn_b_sur[ii,jj]  = cn_beta_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
-                NP_sur[ii,jj]    = neutral_point_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
+                NP_sur[ii,jj]    = neutral_point_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
```
Comment (planes) [inline trunk/SUAVE/Analyses/Stability/AVL.py]: @mclarke2 you need to approve this change.

File: trunk/SUAVE/Attributes/Gases/Air.py [hunk 7]
```diff
@@ -102,7 +102,6 @@ def compute_speed_of_sound(self,T=300.,p=101325.,variable_gamma=False):
             g = self.compute_gamma(T,p)
         else:
             g = 1.4*np.ones_like(T)
-
```
Comment (planes) [inline trunk/SUAVE/Attributes/Gases/Air.py]: Is this a necessary change?

File: trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py [hunk 8]
```diff
@@ -173,68 +173,4 @@ def new_tensor(a):
     
     T = np.resize(T,[n_a,3,3])
     
-    return T
```
Comment (planes) [inline trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py]: You should make this a regression

File: trunk/SUAVE/Analyses/Stability/AVL.py (hunk 6 above)
Comment (WalterMaier) [reply trunk/SUAVE/Analyses/Stability/AVL.py]: There wasn't a change...The commented code below allows the sklearn to work on my own machine.  I assume my version is more up to date than the SUAVE tutorial.

File: regression/scripts/AVL/avl_files/aircraft.avl (hunk 2 above)
Comment (WalterMaier) [reply regression/scripts/AVL/avl_files/aircraft.avl]: Idk...why, but the regression/travis works.... @mclarke2 might be able to comment on the use of it.

File: trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py (hunk 8 above)
Comment (WalterMaier) [reply trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py]: So this is already being covered...are you sure you want another regression?

Comment (planes) [reply trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py]: No leave it alone

File: trunk/SUAVE/Analyses/Stability/AVL.py (hunk 6 above)
Comment (planes) [reply trunk/SUAVE/Analyses/Stability/AVL.py]: Keep those differences locally...have @mclarke2 comment on it

**Pull request merged**